account_invoice_blocking/views/account_invoice.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Acsone SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="account_invoice_form_view">
```

Comment:
@rousseldenis I think that it only makes sense to block supplier invoices so it is right that the field is only visible on supplier invoices.